packages/composite-checkout-wpcom/package.json

```diff
@@ -39,13 +39,15 @@
 		"@automattic/composite-checkout": "file:../composite-checkout",
 		"@emotion/core": "10.0.22",
 		"@emotion/styled": "10.0.23",
-		"@wordpress/data": "^4.9.2",
 		"debug": "4.1.1",
 		"emotion-theming": "10.0.19",
 		"i18n-calypso": "file:../i18n-calypso",
 		"prop-types": "^15.7.2",
-		"react": "^16.8.6",
 		"react-stripe-elements": "^5.1.0"
 	},
+	"peerDependencies": {
+		"@wordpress/data": "^4.9.2",
```

Comment:
If it's helpful, these can be upgraded to 4.10.

Reply:
If anything, we'd want to [make these _more relaxed_](https://github.com/Automattic/wp-calypso/pull/39368#issuecomment-584691961)

> It's also a decent reason to be as compatible as you can, as any change to a peer dep is a major.